src/test/kotlin/com/ampnet/walletservice/controller/CooperativeWithdrawControllerTest.kt

```diff
@@ -237,7 +236,7 @@ class CooperativeWithdrawControllerTest : ControllerTestBase() {
     }
 
     @Test
-    @WithMockCrowdfoundUser(privileges = [PrivilegeType.PWA_WITHDRAW])
+    @WithMockCrowdfoundUser(privileges = [PrivilegeType.PWA_WITHDRAW], coop = coop)
```

Comment:
Sometimes you use `coop = coop` and sometimes `coop = "ampnet"`. Use only one solution and define `coop` value as needed, `cost val` or `protected val`